mod-circulation/src/main/resources/vega/mod-circulation/features/requests.feature

```diff
@@ -1260,4 +1260,138 @@ Feature: Requests tests
     And request requestEntityRequest
     And retry until responseStatus == 422
     When method POST
-    And match $.errors[0].message == blockMessage
\ No newline at end of file
+    And match $.errors[0].message == blockMessage
+
+  Scenario: Test request filtering by call number
+    * def extHoldingsRecordId = call uuid1
+    * def extCallNumber = 'FAT-5355CN'
+    * def extCN = 'FAT5355CN'
+
+    # post an owner
+    * def ownerId = call uuid1
+    * def ownerEntityRequest = read('samples/feefine/owner-entity-request.json')
+    * ownerEntityRequest.id = ownerId
+    Given path 'owners'
+    And request ownerEntityRequest
+    When method POST
+    Then status 201
+
+    # post a material type
+    * def extMaterialTypeId = call uuid1
+    * def extMaterialTypeId2 = call uuid1
+    * def extMaterialTypeName = 'Book'
+    * def extMaterialTypeName2 = 'Text'
```

Comment:
If there's variable named `something2`, there should probably also be `something1`, not just `something`. Also, I thought we only add prefix `ext` to those variables that come from somewhere outside the scenario, it means 'external'. In this case (and many others in this scenario) we're defining them here, so why are we using prefix `ext`?